app/code/Magento/Catalog/view/frontend/web/js/validate-product.js

```diff
@@ -13,7 +13,8 @@ define([
     $.widget('mage.productValidate', {
         options: {
             bindSubmit: false,
-            radioCheckboxClosest: '.nested'
+            radioCheckboxClosest: '.nested',
+            addToCartButtonSelector: '.action.tocart'
```

Comment:
Again, CSS classes cannot be used as a JavaScript logic selectors. Please check examples in link I posted.

Reply:
@orlangur  magento itself using that same class into add to cart js so i am using that
https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Catalog/view/frontend/web/js/catalog-add-to-cart.js#L23

let me know if still you want to change it